test/test_const_value_dist.jl

```diff
@@ -77,4 +77,23 @@ using Random
     @test @inferred(mode(univariate_cvd)) == 42
 
     @test @inferred(eltype(univariate_cvd)) == Int64
+    
+    μ1, μ2 = rand(2)
+    cvd_from_named_tuple = @inferred(ConstValueDist((a=μ1, b=μ2)))
```

Comment:
The constructor of a `ConstValueDist` when using a named tuple may not be strict enough. For example, something like
```
const_value_dist_from_scalars = ConstValueDist((a=0.0, b=0.0))
const_value_dist_from_normals = ConstValueDist((a=Normal(), b=Normal()))

# Works as expected
pdf(const_value_dist_from_scalars, (a=0.0, b=0.0)) == Inf

# Not sure about these two pdf evaluations
pdf(const_value_dist_from_normals, (a=0, b=0)) == 0.0
pdf(const_value_dist_from_normals, (a=Normal(), b=Normal())) == Inf
```
Not sure what it means to evaluate a distribution at a distribution. Is this intended to work as shown above?

Reply:
The value wrapped in `ConstValueDist` should (typically) *not* include distributions. A `ConstValueDist` is a dirac-style-distribution (like the new `Distributions.Dirac`, we'll have to see if we can switch to using that in the future, possibly with some additions).